internal/backend/sessions/sessionworkflows/syscalls.go

```diff
@@ -57,7 +57,7 @@ func (w *sessionWorkflow) syscall(ctx context.Context, args []sdktypes.Value, kw
 	}
 }
 
-func (w *sessionWorkflow) fake(args []sdktypes.Value, kwargs map[string]sdktypes.Value) (sdktypes.Value, error) {
+func (w *sessionWorkflow) fake(_ context.Context, args []sdktypes.Value, kwargs map[string]sdktypes.Value) (sdktypes.Value, error) {
```

Comment:
What is `fake` for? What does it do?

Reply:
i did it a long while ago as am exploration of testing. it allows to tell autokitteh to run specific activities with predetermined values instead of actually invoking the activity.